lib/ExpensiMark.js

```diff
@@ -195,15 +200,25 @@ export default class ExpensiMark {
                 // We also want to capture a blank line before or after the quote so that we do not add extra spaces.
                 // block quotes naturally appear on their own line. Blockquotes should not appear in code fences or
                 // inline code blocks. A single prepending space should be stripped if it exists
-                process: (textToProcess, replacement) => {
+                process: (textToProcess, replacement, shouldKeepWhitespace = false) => {
                     const regex = new RegExp(
-                        /\n?^&gt; *(?! )(?![^<]*(?:<\/pre>|<\/code>))([^\v\n\r]+)\n?/gm,
+                        /^&gt; *(?! )(?![^<]*(?:<\/pre>|<\/code>))([^\v\n\r]+)/gm,
                     );
+                    if (shouldKeepWhitespace) {
+                        return textToProcess.replace(regex, g1 => replacement(g1, shouldKeepWhitespace));
```

Comment:
Can we skip calling `modifyTextForQuote` in this case?

Reply:
I'm not sure, after skipping `modifyTextForQuote` 13 test fails since this method is trimming spaces and newlines in strings. I didn't want to break the existing behavior so I've added a new one but behind the `shouldKeepWhitespace` flag

Reply:
Okay, thanks for explanation.